### DISCUSSION TITLE
[Enhancement] Autolinking account links in commit messages

### DISCUSSION THREAD

**Discussion by kuraga** [community/community#74845] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Good day!

Account links (such as `https://github.com/kuraga`) are autolinked but aren't converted to @kuraga .

Also mentions (such as `@kuraga`) [are not autolinked](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls) in commit messages.

Proposition: convert the first and autolink the latter.

Thanks!

**Comment by happy-spark** [comment]
In the Commit message, if the link and text are stuck together with no space between them, it might not be an automatic link.

> I used a translator, so the context might be a little weird. :)

**Reply by kuraga** [reply]
Updated.
